DSharpPlus/Entities/Voice/DiscordVoiceState.cs

```diff
@@ -16,159 +17,118 @@ public class DiscordVoiceState
     /// Gets ID of the guild this voice state is associated with.
     /// </summary>
     [JsonProperty("guild_id", NullValueHandling = NullValueHandling.Ignore)]
-    internal ulong? GuildId { get; set; }
+    internal ulong? GuildId { get; init; }
 
     /// <summary>
     /// Gets the guild associated with this voice state.
     /// </summary>
     [JsonIgnore]
-    public DiscordGuild Guild
-        => this.GuildId != null ? this.Discord.Guilds[this.GuildId.Value] : this.Channel?.Guild;
+    public DiscordGuild? Guild => this.GuildId is not null ? this.Discord.Guilds[this.GuildId.Value] : this.Channel?.Guild;
 
     /// <summary>
     /// Gets ID of the channel this user is connected to.
     /// </summary>
     [JsonProperty("channel_id", NullValueHandling = NullValueHandling.Include)]
-    internal ulong? ChannelId { get; set; }
+    internal ulong? ChannelId { get; init; }
 
     /// <summary>
     /// Gets the channel this user is connected to.
     /// </summary>
     [JsonIgnore]
-    public DiscordChannel Channel
-        => this.ChannelId != null && this.ChannelId.Value != 0 ? this.Discord.InternalGetCachedChannel(this.ChannelId.Value) : null;
+    public DiscordChannel? Channel => (this.ChannelId.HasValue && this.ChannelId.Value != 0) ? this.Discord.InternalGetCachedChannel(this.ChannelId.Value) : null;
 
     /// <summary>
     /// Gets ID of the user to which this voice state belongs.
     /// </summary>
     [JsonProperty("user_id", NullValueHandling = NullValueHandling.Ignore)]
-    internal ulong UserId { get; set; }
+    internal ulong UserId { get; init; }
 
     /// <summary>
     /// Gets the user associated with this voice state.
     /// <para>This can be cast to a <see cref="DiscordMember"/> if this voice state was in a guild.</para>
     /// </summary>
     [JsonIgnore]
-    public DiscordUser User
-    {
-        get
-        {
-            DiscordUser? usr = null as DiscordUser;
-
-            if (this.Guild != null)
-            {
-                usr = this.Guild._members.TryGetValue(this.UserId, out DiscordMember? member) ? member : null;
-            }
-
-            if (usr == null)
-            {
-                usr = this.Discord.GetCachedOrEmptyUserInternal(this.UserId);
-            }
-
-            return usr;
-        }
-    }
+    public DiscordUser User => this.Guild is not null && this.Guild._members.TryGetValue(this.UserId, out DiscordMember? member)
+        ? member
+        : this.Discord.GetCachedOrEmptyUserInternal(this.UserId);
```

Comment:
i don't like this formatting - fwiw, the rule for methods is 'expression body if they're a single line', iirc

Reply:
I can format them to be on the same line :^)

Reply:
no.